testsuite/MDAnalysisTests/core/test_groups.py

```diff
@@ -871,6 +871,7 @@ def succeeding_pairs(left, right):
         # Do succeeding pair actually succeed
         for level in ('atoms', 'residues', 'segments'):
             # Groups
+            # pylint: disable=unsubscriptable-object
```

Comment:
Nitpicking: the spacing between the comment sign (`#`) and `pylint:` is inconsistent. I vote for a single space, as it is here and in pylint's docs.